Resources/Prototypes/_DV/Entities/Mobs/Species/harpy.yml

```
@@ -6,6 +6,10 @@
   abstract: true
   components:
   - type: HarpySinger
+  - type: Flight # EE - Flight
+    isLayerAnimated: true
+    layer: "/Textures/Mobs/Customization/Harpy/harpy_wings.rsi"
```

Wrong path; Add `_EE` to make flapping animation work
```suggestion
    layer: "/Textures/_EE/Mobs/Customization/Harpy/harpy_wings.rsi"
```